### DISCUSSION TITLE
Watermark

### DISCUSSION THREAD

**Discussion by BradSena** [xtremestuff/protune-transforms#5] [General]
When using the LUTs I have a "DAVINCI Studio"  watermark over my image. Why is that? Aren't the Luts compatible with free version of Da Vinci?

**Comment by RikshaDriver** [comment]
> When using the LUTs I have a "DAVINCI Studio"  watermark over my image. Why is that? Aren't the Luts compatible with free version of Da Vinci?

These are DCTL transforms, not LUTs. The free version of Resolve has a watermark when using DCTLs as that is a Studio feature. Having said that, you can still use the ACES Input Transform DCTLs with the free version of Resolve without any Studio watermarks.

**Reply by BradSena** [reply]
Thanks for your quick answer, I'm not familiar with all this yet.
I think I managed to make it work, here are my settings, can you confirm they are OK?

![Capture d’écran 2023-05-18 à 14 04 05](https://github.com/xtremestuff/protune-transforms/assets/93638410/a4a965cc-10c8-427f-8e11-8374e9782c40)

Why does my histogram looks completely cropped on the right?

![Capture d’écran 2023-05-18 à 14 03 38](https://github.com/xtremestuff/protune-transforms/assets/93638410/a1a791e5-3902-477b-969d-fa134cbc0650)

Here is the same frame in davinci color science instead of ACES, I'm I doing something wrong or is it to be expected?


![Capture d’écran 2023-05-18 à 14 08 18](https://github.com/xtremestuff/protune-transforms/assets/93638410/cf814683-d0a6-481d-a919-c77483ce653c)

**Reply by RikshaDriver** [reply]
ACES has Tone Mapping applied, your other setting likely doesn't - hence the difference.

You should look at the scopes, not the histogram.
